libparsec/src/cli/mod.rs

```diff
@@ -0,0 +1,9 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+mod list_devices;
+
+pub use list_devices::list_devices;
+
+const GREEN: &str = "\x1B[92m";
+const RESET: &str = "\x1B[39m";
+const YELLOW: &str = "\x1B[33m";
```

Comment:
For writing a tui, I would recommend using one of:

- [tui](https://crates.io/crates/tui): Use `crossterm` as backend but can use `termion`. 
- [crossterm](https://crates.io/crates/tui): Support all platform (Window, Unix).
- [termion](https://crates.io/crates/termion): Seems to support only Unix.

Reply:
It depends if I need more. For the moment it's only `3` lines